src/backend/distributed/planner/multi_explain.c

```diff
@@ -199,6 +201,21 @@ CitusExplainScan(CustomScanState *node, List *ancestors, struct ExplainState *es
 		return;
 	}
 
+	if (SkipConstraintValidation || (AlterTableInProgress() && IsCitusInternalBackend()))
```

Comment:
AFAIK, constraints could be created/validated only by using ALTER TABLE.
Do we need to have a `SkipConstraintValidation` check here? It looks redundant. Also it probably might produce unwanted side effects like explain not working correctly for normal queries when `citus.skip_constraint_validation` GUC is set.

Reply:
You are right, `SkipConstraintValidation` seems wrong (which I suggested to Emel).

 I think we don't even need `IsCitusInternalBackend`, we can just leave `AlterTableInProgress`. Because, even on the coordinator, the EXPLAIN is just useless as we skip the constraint checks anyway.

What do you think?